pkg/sanity/controller.go

```diff
@@ -1118,6 +1118,21 @@ var _ = DescribeSanity("Controller Service", func(sc *SanityContext) {
 
 		It("should return appropriate values (no optional values added)", func() {
 
+			By("getting node information")
+			ni, err := n.NodeGetInfo(
+				context.Background(),
+				&csi.NodeGetInfoRequest{})
+			Expect(err).NotTo(HaveOccurred())
+			Expect(ni).NotTo(BeNil())
+			Expect(ni.GetNodeId()).NotTo(BeEmpty())
+
+			var accReqs *csi.TopologyRequirement
+			if ni.AccessibleTopology != nil {
+				accReqs = &csi.TopologyRequirement{
+					Requisite: []*csi.Topology{ni.AccessibleTopology},
```

Comment:
This assumes that the storage backend can support a one-node topology, but may not be the case for something that requires say a min of 3 nodes for replication.

I'm unsure what would be a great way to support that in the test though.  This is probably ok for now, but it might be good to add a comment about the assumption.

Reply:
csi-sanity essentially simulates a CO with a single node, so I think this code change is consistent with the rest of the testing.

@alexanderKhaustov can you add a short comment here that explains that?

Reply:
Sorry for the delay
I'm not sure I understand the potentially problematic scenario.
The SP is returning topology information for a node, but then it doesn't support provisioning a volume with this topology, because of a requirement (existing only externally to csi plugin/spec?) that a volume can only be created if there are no less than 3 nodes with some specific topology?
If that is the case then it seems that such an external peculiarity is hardly within the scope of csi-test, is it not? (csi-test aside, it seems that such a plugin would also perform unexpectedly for a CO end-user, would it not?)

Reply:
@alexanderKhaustov: I agree. Let me try to propose a comment that you can put above the new code and then if @msau42 has no additional comments we can merge it:

> csi-sanity testing emulates the behavior of a CO with a single node. The name and topology of that node is provided by the driver itself. Here we ensure that the new volume gets provisioned on that node, because otherwise staging it on the node later in the test might fail.

Reply:
I think it would be good to add a note that this test does not support storage backends that require more than 1 node.

Reply:
I've checked the spec more thoroughly, seems like there are no requirements for the setups that a SP MUST support. So I added comments about test assumptions.

As an aside, with regard to the topology returned by NodeGetInfo the spec suprisingly states that 
> COs MAY use this [topology] information ... when scheduling workloads

(https://github.com/container-storage-interface/spec/blob/master/spec.md#nodegetinfo)
rather than MUST use which seems more reasonable. Do you know any reason for that?

Reply:
I don't know why topology information is allowed to be ignored.